core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -443,6 +450,10 @@ public boolean isFutureAndCurrentLocationSet() {
     return futureAndCurrentLocationSet;
   }
 
+  public boolean isOperationIdAndCurrentLocationSet() {
```

Comment:
Check `TabletManagementIterator.computeTabletManagementActions`and [TabletGroupWatcher](https://github.com/apache/accumulo/blob/elasticity/server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java#L252) to see how to wire this up.

Reply:
Added in e79f423